src/devices/HardwareMonitor/OpenHardwareMonitor.cs

```diff
@@ -657,6 +670,11 @@ public Sensor(ManagementObject instance, string name, string identifier, string?
             /// </summary>
             public SensorType SensorType { get; }
 
+            /// <summary>
+            /// The instance ID. Used to retrieve a new value for the same sensor (repeatedly obtaining the same value from a sensor instance returns the same value)
+            /// </summary>
+            internal int InstanceId { get; }
```

Comment:
Looks like some of the changes on this binding are not related to just the addition of the configurations. Can we for now only include the changes related to netstandard2.0 configs and then send out a follow up for the HardwareMonitor changes? It just m akes it easier to audit these type of changes in the future.